crates/core/src/contracts/tests/mod.rs

```diff
@@ -0,0 +1,27 @@
+use reth_primitives::U256;
+use starknet::core::types::BlockId;
+use starknet::providers::SequencerGatewayProvider;
+use starknet_crypto::FieldElement;
+
+use crate::client::api::KakarotStarknetApi;
+use crate::client::constants::STARKNET_NATIVE_TOKEN;
+use crate::client::tests::init_testnet_client;
+use crate::contracts::erc20::Erc20Contract;
+
+#[tokio::test]
+async fn test_balance_of() {
+    // Given
+    let client = init_testnet_client();
+    let starknet_native_token_address = FieldElement::from_hex_be(STARKNET_NATIVE_TOKEN).unwrap();
+    let erc20 = Erc20Contract::<SequencerGatewayProvider>::new(starknet_native_token_address);
+
+    let target_address =
+        FieldElement::from_hex_be("0x05590dc5e5bddf4b334e31713f8caf820f58f6393189e33aeae891ba8534aeb6").unwrap();
```

Comment:
whose address is this? can we comment above this hardcoded value?

Reply:
changed to ACCOUNT_ADDRESS